### ISSUE TITLE
Feature request for silent downloads

### ISSUE THREAD

**Issue by nkmathew** [elmoiv/redvid#8]
Hey Khaled,

Thanks for the making this. Do you think you could add an option for disabling the progress messages so we can clean logs? Like a silent flag in most command line applications

Also I have a use case where I'd like to only mirror a reddit video if it's not too big e.g if the video is over 20MB don't download it. Is this something that can be implemented?

**Comment by elmoiv** [comment]
Thanks @nkmathew for these great ideas. I am busy today, so I will try to make this possible ASAP.

Stay tuned :)

**Comment by elmoiv** [comment]
Silent download will work only if `max_q` or `min_q` is set `True`. If not, you will be prompted to quality query to select desired one.
So be sure to check on of the mentioned attributes before downloading.

Updated version will be soon :)

**Comment by elmoiv** [comment]
Published v1.0.9:
  * Added the ability to disable logging.
  * Maximum video size can be set.
  * Maximum video duration can be set.

To download videos without logging (silent mode):
```python
from redvid import Downloader

reddit = Downloader()

# Due to silent mode, user quality query won't be visible so redvid
# automatically downloads maximum quality unless minimum is set
reddit.log = False
reddit.url = 'https://www.reddit.com/r/PublicFreakout/comments/ho5ml9/former_judges_michael_conahan_and_mark_ciavarella/'
reddit.download()
```


To set a maximum size that is not allowed to be exceeded (like your 20 MB example):
```python
from redvid import Downloader

reddit = Downloader()

# Set max size to 20 MB
# Size is passed in BYTES
reddit.max_s = 20 * (1 << 20) 

# The next video at max 1080p is 125 MB
reddit.url = 'https://www.reddit.com/r/PublicFreakout/comments/ho5ml9/former_judges_michael_conahan_and_mark_ciavarella/'
reddit.download()
```
Now if the video exceeds 20 MB, nothing will be downloaded and function will return error number corresponding to size which equals 0.

Hope you find redvid helpful all the time :)
